modules/storage.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable class-methods-use-this */
+export class Storage {
+  constructor() {
+    return null;
+  }
+
+  storeBooks(books) {
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  fetchBooks() {
+    const books = JSON.parse(localStorage.getItem('books'));
+    if (books) return books;
+    return [];
+  }
```

Comment:
- [ ] [OPTIONAL] it would be better if you use es6 `arrow function` instead of `normal method` in order to practice working with both